FlowCrypt/Controllers/Compose/ComposeViewControllerInput.swift

```diff
@@ -31,8 +32,42 @@ struct ComposeMessageInput: Equatable {
 
     var isQuote: Bool {
         switch type {
-        case .idle: return false
-        case .quote: return true
+        case .idle:
+            return false
+        case .quote:
+            return true
+        }
+    }
+
+    var isIdle: Bool {
+        !isQuote
+    }
+
+    var isReply: Bool {
+        switch type {
+        case .idle:
+            return false
+        case .quote(let messageQuoteInfo):
+            switch messageQuoteInfo.quoteType {
+            case .reply:
+                return true
+            case .forward:
+                return false
+            }
+        }
+    }
+
+    var isForward: Bool {
+        switch type {
+        case .idle:
+            return false
+        case .quote(let messageQuoteInfo):
+            switch messageQuoteInfo.quoteType {
+            case .reply:
+                return false
+            case .forward:
+                return true
+            }
```

Comment:
Current implementation with 3 different properties `isIdle`, `isReply` and `isForward` looks a bit complicated.

I think updating `InputType` to look like this will make code more clear
```swift
enum InputType: Equatable {
    case idle
    case reply(MessageQuoteInfo)
    case forward(MessageQuoteInfo)
}
```
Then `isQuote` will be just
```swift
var isQuote: Bool {
    return type != .idle
 }
```
and we won't need complex `switch` for checking other types.

Reply:
Completely agree with solution 
```
enum InputType: Equatable {
    case idle
    case reply(MessageQuoteInfo)
    case forward(MessageQuoteInfo)
}
```
I thought it was done intentionally. Will apply this changes

Reply:
Previously we didn't need to separate `forward` and `reply` for this screen, so `quote` type was enough.
But for this functionality and for possible future features it'll be better to have separate `InputType` cases for each type.